documentation/domains/Cluster.json

```diff
@@ -86,6 +86,22 @@
         }
       }
     },
+    "BaseServerPod": {
```

Comment:
It doesn't look like this belongs in the cluster scope. Also, the name 'BaseServerPod' looks internal/unintended.

Reply:
Renamed to `IntrospectorPod`.